client/src/components/Tea/TeaList/TableFilter.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import Datalist from "../../Datalist";
+
+export const TableFilter = props => {
+  return (
+    <div className="columns is-pulled-right">
+      <form onSubmit={() => props.handleFilterClick(event)}>
+        <div className="field has-addons">
+          <p className="control">
```

Comment:
Why paragraphs?

Reply:
that's how Bulma has it set up. i'll see if i can swap this for divs.